operator/controllers/seldondeployment_prepackaged_servers.go

```diff
@@ -299,7 +286,7 @@ func (pi *PrePackedInitialiser) addModelDefaultServers(pu *machinelearningv1.Pre
 		}
 	}
 
-	machinelearningv1.SetImageNameForPrepackContainer(pu, c, serverConfig)
+	c.Image = serverConfig.PrepackImageName(mlDepSepc.Protocol, pu)
```

Comment:
Shouldn't we check here if there was an image previously set in `c`?

Reply:
yes will check